assets/wire/tests/WireApp.spec.tsx

```diff
@@ -125,6 +125,20 @@ describe('shortHighlightedtext', () => {
         expect(output).toEqual(`On olemassa myös sellaisia ihmisiä, joiden silmissä
         jokainen parisuhde muuttuu <span class="es-highlight">mukaanalkuhuuma</span> <span class="es-highlight">kestää</span> yleensä 1–2 vuotta...`);
     });
-       
+
+    it('returns highlight text condition with more tags', () => {
+        const html = '<ul><li><span class="es-highlight">foo</span></li><li><span class="es-highlight">foo</span></li><li>bar</li><li><span class="es-highlight">foo</span></li></ul>';
+        const maxLength = 40;
+        const output = shortHighlightedtext(html, maxLength);
+        expect(output).toEqual('<span class="es-highlight">foo</span>...');
```

Comment:
would be good to actually output here:
```html
<span class="es-highlight">foo</span> <span class="es-highlight">foo</span> bar <span class="es-highlight">foo</span>
```
just not sure if we should have line breaks there in between or just spaces

Reply:
spaces would be probably better or some `...`

Reply:
I see, however I suppose if we handle that situation, it will break in a lengthy paragraph?

Reply:
yep but you will see more matches which is good, while it's not that long still. showing just the first match is not enough imo

Reply:
I tried to handle that case, but seems liken other cases are failed